src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -23,17 +23,7 @@
 #endif
 
 #if IBV_HW_TM
-#  if HAVE_INFINIBAND_TM_TYPES_H
```

Comment:
seems we ignore the check AC_CHECK_HEADERS([infiniband/tm_types.h])
shall we remove it from configure? or inset IBV_HW_TM if the check failed?